proto/attest.proto

```diff
@@ -125,10 +125,15 @@ message ContainerState {
   string image_reference = 1;
   bytes image_digest = 2;
   RestartPolicy restart_policy = 3;
+  string image_id = 4;
```

Comment:
Having `image_id` be a string is good, as the data will look like `sha256:<hex>`. For similar reasons we should make the `image_digest` also be a string (as it has the same format).

Also, can you add a comment for these two fields explaining:
  - their format
  - how they are different

Reply:
as this isn't yet used, should we do a back incompat change? or would you prefer a deprecated comment and new field?

Reply:
Backwards incompatible is fine, also [changing a `bytes` field to a `string` field is backwards compatible](https://developers.google.com/protocol-buffers/docs/overview#updating) in general (provided that the bytes messages have always contained valid UTF8. No really relevent here, but an interesting fact.

Reply:
cool, TIL!